test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class StaticPagesControllerTest < ActionController::TestCase
+  test "should get home" do
+    get :home
+    assert_response :success
+    assert_select "title", "Home | Ruby on Rails Tutorial Sample App"
+  end
+
+  test "should get help" do
+    get :help
+    assert_response :success
+    assert_select "title", "Help | Ruby on Rails Tutorial Sample App"
+  end
+
+  test "should get about" do
+    get :about
+    assert_response :success
+    assert_select "title", "About | Ruby on Rails Tutorial Sample App"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)